broker-daemon/models/block-order.js

```diff
@@ -263,6 +276,30 @@ class BlockOrder {
 
     return new this({ id, marketName, side, amount, price, timeInForce, status })
   }
+
+  /**
+   * Grab a block order from a given sublevel
+   *
+   * @param {Sublevel} store block order sublevel store
+   * @param {String} blockOrderId
+   * @return {BlockOrder} BlockOrder instance
+   * @throws {Error} store is null
+   * @throws {BlockOrderNotFoundError} block order could not be found
+   */
+  static async fromStore (store, blockOrderId) {
+    if (!store) throw new Error('[BlockOrder#fromStore] No leveldb store is defined')
```

Comment:
[nit] I don't think there is a good reason to reference leveldb in the error message